src/PrestaShopBundle/Form/Admin/ShopParameters/General/PreferencesType.php

```diff
@@ -0,0 +1,197 @@
+                'choices_as_values' => true,
+                'choices'  => array(
+                    'No' => false,
+                    'Yes' => true,
+                ),
+                'choice_translation_domain' => 'Admin.Global',
+            ))
+            ->add('price_round_mode', ChoiceType::class, array(
+                'choices_as_values' => true,
+                'choices'  => array(
+                    'Round up away from zero, when it is half way there (recommended)' =>
+                        $configuration->get('PS_ROUND_HALF_UP'),
+                    'Round down towards zero, when it is half way there' =>
+                        $configuration->get('PS_ROUND_HALF_DOWN'),
+                    'Round towards the next even value' =>
+                        $configuration->get('PS_ROUND_HALF_EVEN'),
+                    'Round towards the next odd value' =>
+                        $configuration->get('PS_ROUND_HALF_ODD'),
+                    'Round up to the nearest value' =>
+                        $configuration->get('PS_ROUND_UP'),
+                    'Round down to the nearest value' =>
+                        $configuration->get('PS_ROUND_DOWN'),
+                ),
+            ))
+            ->add('price_round_type', ChoiceType::class, array(
+                'choices_as_values' => true,
+                'choices'  => array(
+                    'Round on each item' => Order::ROUND_ITEM,
+                    'Round on each line' => Order::ROUND_LINE,
+                    'Round on the total' => Order::ROUND_TOTAL,
+                ),
+            ))
+            ->add('price_display_precision', IntegerType::class, array(
+                'attr' => [
+                    'min' => 0,
+                ],
+            ))
+            ->add('display_suppliers', ChoiceType::class, array(
+                'choices_as_values' => true,
+                'choices'  => array(
+                    'No' => false,
+                    'Yes' => true,
+                ),
+                'choice_translation_domain' => 'Admin.Global',
+            ))
+            ->add('display_best_sellers', ChoiceType::class, array(
+                'choices_as_values' => true,
+                'choices'  => array(
+                    'No' => false,
+                    'Yes' => true,
+                ),
+                'choice_translation_domain' => 'Admin.Global',
+            ))
+            ->add('multishop_feature_active', ChoiceType::class, array(
+                'choices_as_values' => true,
+                'choices'  => array(
+                    'No' => false,
+                    'Yes' => true,
+                ),
+                'choice_translation_domain' => 'Admin.Global',
+            ))
+            ->add('shop_activity', ChoiceType::class, array(
+                'choices_as_values' => true,
+                'choices'  => array(
+                    '-- Please choose your main activity --' => 0,
```

Comment:
what you are doing here is create a group => https://symfony.com/doc/3.4/reference/forms/types/choice.html#grouping-options